cpp/modmesh/view/RManager.cpp

```diff
@@ -217,50 +206,54 @@ void RManager::setUpMenu()
         use_fps_camera->setCheckable(true);
         use_orbit_camera->setChecked(true);
 
-        m_cameraMenu->addAction(use_orbit_camera);
-        m_cameraMenu->addAction(use_fps_camera);
+        m_viewMenu->addAction(use_orbit_camera);
+        m_viewMenu->addAction(use_fps_camera);
     }
 
-    {
-        m_appMenu = m_mainWindow->menuBar()->addMenu(QString("App"));
-
-        this->addApplication(QString("sample_mesh"));
-        this->addApplication(QString("linear_wave"));
-        this->addApplication(QString("bad_euler1d"));
-    }
-
-    {
-        QAction * params = new RAction(
-            QString("Parameters"),
-            QString("Runtime parameters"),
-            []()
-            {
-                static int64_t int64V = 5566;
-                static double doubleV = 77.88;
-                auto params = createParameters();
-                addParam(params, "global.a.b.int64_foo", &int64V);
-                addParam(params, "global.a.b.double_bar", &doubleV);
-                openParameterView(params);
-            });
-        m_fileMenu->addAction(params);
-    }
+    m_oneMenu = m_mainWindow->menuBar()->addMenu(QString("One"));
+    m_oneMenu->addAction(new RAppAction(
+        QString("Load euler1d"),
+        QString("Load euler1d"),
+        QString("modmesh.app.euler1d")));
+
+    m_meshMenu = m_mainWindow->menuBar()->addMenu(QString("Mesh"));
+    m_meshMenu->addAction(new RAppAction(
+        QString("Load mh_3dmix"),
+        QString("Load mh_3dmix"),
+        QString("modmesh.app.mh_3dmix")));
+    m_meshMenu->addAction(new RAppAction(
+        QString("Load naca"),
+        QString("Load naca"),
+        QString("modmesh.app.naca")));
+
+    m_addonMenu = m_mainWindow->menuBar()->addMenu(QString("Addon"));
+    this->addApplication(QString("sample_mesh"));
+    this->addApplication(QString("linear_wave"));
+    this->addApplication(QString("bad_euler1d"));
+
+    m_windowMenu = m_mainWindow->menuBar()->addMenu(QString("Window"));
```

Comment:
We should add code to manage the sub-windows, but in this PR I do not plan to add the code yet.  Only add the menubar.

Reply:
Agree, currently sub-windows are opened, the sub-windows will overlap each other which is not good for user experience, we can open an issue to track it!